met-api/src/met_api/services/object_storage_service.py

```diff
@@ -0,0 +1,50 @@
+
+"""Service for object storage management."""
+import os
+import uuid
+from typing import List
+import requests
+from markupsafe import string
+from aws_requests_auth.aws_auth import AWSRequestsAuth
+from met_api.config import _Config
+from met_api.schemas.document import Document
+
+
+class ObjectStorageService:
+    """Object storage management service."""
+
+    @staticmethod
+    def get_url(filename: string):
+        """Get the object url."""
+        return f'https://{_Config.OSS_S3_HOST}/{_Config.OSS_S3_BUCKET}/{filename}' if filename else ''
+
+    def get_auth_headers(self, documents: List[Document]):
+        """Get the s3 auth headers or the provided documents."""
+        if(_Config.OSS_S3_ACCESS_KEY_ID is None or
+            _Config.OSS_S3_SECRET_ACCESS_KEY is None or
+            _Config.OSS_S3_HOST is None or
+            _Config.OSS_S3_BUCKET is None
+           ):
+            return {'status': 'Configuration Issue',
+                    'message': 'accesskey is None or secretkey is None or S3 host is None or formsbucket is None'}, 500
```

Comment:
ideally throw an exception here since this is configuration missing ;but ok for now

Reply:
noted